### PR TITLE
Tune dev resources / autoscaling

### PR DESCRIPTION
**Pull request by francardoso93**

### CODE REVIEW

File: helm/values-dev.yaml [hunk 1]
```diff
@@ -6,7 +6,15 @@ service:
     metricsMonitor:
       enable: false
     healthCheck:
-      enable: false
+      enable: "true"
+      liveness: 
+        initialDelaySeconds: 15
+        periodSeconds: 5
```
Comment (blucas) [inline helm/values-dev.yaml:12]:
```suggestion
        periodSeconds: 5
        timeoutSeconds: 1
        failureThreshold: 3
        successThreshold: 1
```

Let's be explicit about these settings so there is no confusion as to what each one is configured with.  The values I've set for `timeout`, `failure`, and `success` are the defaults kubernetes would set.

File: helm/values-dev.yaml [hunk 2]
```diff
@@ -6,7 +6,15 @@ service:
...
+      readiness:
+        initialDelaySeconds: 15
+        periodSeconds: 12
+        timeoutSeconds: 10
+        failureThreshold: 10
```
Comment (blucas) [inline helm/values-dev.yaml:20]:
```suggestion
        failureThreshold: 10
        successThreshold: 1
```
Let's be explicit about these settings so there is no confusion as to what each one is configured with. The value I've set for `success` is the default kubernetes would set.

**Review by blucas — CHANGES_REQUESTED** [review]

**Review by blucas — APPROVED** [review]

**Review by ericssontadashi — APPROVED** [review]

**Comment by francardoso93** [comment]
I'm waiting on definitions of changing the instance types for dev and staging before approving this

**Comment by francardoso93** [comment]
> I'm waiting on definitions of changing the instance types for dev and staging before approving this

https://github.com/elastic-ipfs/infrastructure/pull/191

**Pull request merged**